doc/plugins.dox

```diff
@@ -186,59 +195,143 @@ Let's start with the very basic headers and add more later if necessary.
 @a geanyplugin.h includes all of the Geany API and also the necessary GTK header files,
 so there is no need to include @a gtk/gtk.h yourself.
 
-@note
-@a plugindata.h contains the biggest part of the plugin API and provides some basic macros.
-
-Then you should define two basic variables which will give access to data fields
-provided by the plugin API.
-@code
-GeanyPlugin			*geany_plugin;
-GeanyData			*geany_data;
-@endcode
+@note @a plugindata.h contains the biggest part of the plugin API and provides some basic macros.
```

Comment:
Agreed.  Also plugins should only include `<geanyplugin.h>` so there's no point in confusing the user further IMO.  

A bit unrelated to this PR though, but as you're touching this line it could be the opportunity to get rid of this.